jekyll/_cci2/artifacts.md

```diff
@@ -165,12 +166,31 @@ curl https://circleci.com/api/v1.1/project/:vcs-type/:username/:project/:build_n
 <artifacts.txt xargs -P4 -I % wget %?circle-token=$CIRCLE_TOKEN
 ```
 
-The `curl` command fetches all artifact details for a build
-and pipes it to `grep` to extract the URLs.
-The results are saved to the `artifacts.txt` file.
-Finally, `xargs` reads the file
-and downloads each artifact to the current directory.
+Placeholder   | Meaning                                                                       |
+--------------|-------------------------------------------------------------------------------|
+`:your_token` | The personal API token you created above.
+`:vcs-type`   | The version control system (VCS) you are using. Either `github` or `bitbucket`.
+`:username`   | The VCS project account username for the target project. Located at the top left of the screen in the CircleCI application.
+`:project`    | The name of the target project.
```

Comment:
We might want to mention that word repo/repository in here as that's what it is on GitHub's end.

Reply:
Ha, for some reason I thought that BB called their repos something else, but TIL wooo :)